packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -598,11 +598,11 @@ class UpdatePackagesCommand extends FlutterCommand {
       }
     }
 
-    for (_DependencyLink path in paths) {
+    for (_DependencyLink? path in paths) {
       final StringBuffer buf = StringBuffer();
       while (path != null) {
         buf.write(path.to);
-        path = path.from!;
+        path = path.from;
```

Comment:
LOL, I'm guessing if you tried to run this before this change you'd get a _CastError, good catch!

Reply:
(this is a maintainer tool for diagnosing version solving problems, so not often run)

Reply:
Hehe. I was very curious about how this piece of code was running before...